### ISSUE TITLE
Training is overfitting

### ISSUE THREAD

**Issue by aylek** [jiang-du/BlazePose-tensorflow#16]
Training the joints model is overfitting to training data. Training is converging, but validation accuracy hovers around 10000% according to mse training metric. Is this a known issue? If so, any guidance on how to address it?
